From d20929e0422398f34a241a11d9183fc7750c9f4e Mon Sep 17 00:00:00 2001
From: Ludovic Desroches <ludovic.desroches@atmel.com>
Date: Wed, 19 Sep 2012 10:02:32 +0200
Subject: ARM: at91: missing header file for rtc-at91rm9200.c

Missing asm/io.h inclusion causing issue with __raw_readl and
__raw_writel.

Signed-off-by: Ludovic Desroches <ludovic.desroches@atmel.com>
---
 drivers/rtc/rtc-at91rm9200.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/rtc/rtc-at91rm9200.c b/drivers/rtc/rtc-at91rm9200.c
index dc474bc..f02acb0 100644
--- a/drivers/rtc/rtc-at91rm9200.c
+++ b/drivers/rtc/rtc-at91rm9200.c
@@ -28,6 +28,7 @@
 #include <linux/ioctl.h>
 #include <linux/completion.h>
 
+#include <asm/io.h>
 #include <asm/uaccess.h>
 
 #include <mach/at91_rtc.h>
-- 
1.8.0.197.g5a90748

